client/state/selectors/has-user-purchased-a-plan.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Naming nit: This means the user has a site with an _active_ plan, is that correct?

Maybe this selector should be `userHasAnyPaidPlan` or something along those lines?

Reply:
You're totally right, renamed in https://github.com/Automattic/wp-calypso/pull/19421/commits/f08d29e1f7ea26202d16bdfc0d003b8365b92b80.